doc_classes/TrainDoor.xml

```diff
@@ -0,0 +1,131 @@
+			When the door closing button is pressed, a buzz sound is activated automatically, and when released, the doors close.
+		</member>
+		<member name="blocked" type="bool" setter="set_door_blocked" getter="get_door_blocked" default="false">
+			Is there a door lock
+		</member>
+		<member name="close/delay" type="float" setter="set_door_close_delay" getter="get_door_close_delay" default="0.0">
+			Door closing delay, in seconds.
+		</member>
+		<member name="close/method" type="int" setter="set_door_close_method" getter="get_door_close_method" default="0">
+			Determines method of closing doors.
+			The following values are valid:
+			Passenger - default type when no definition is provided; doors are manually closed, ignoring any remote commands
+			AutomaticCtrl - doors operate automatically
+			DriverCtrl - doors are controlled by the driver of the train and only respond to remote commands
+			Conductor - doors are controlled by the conductor of the train and only respond to remote commands
+			Mixed - doors can be closed both manually and remotely
+		</member>
+		<member name="close/speed" type="float" setter="set_close_speed" getter="get_close_speed" default="0.0">
+			Animation speed for closing doors
+		</member>
+		<member name="closure/warning" type="bool" setter="set_door_closure_warning" getter="get_door_closure_warning" default="false">
+			Only a buzzer (used before closing the doors, like in EN57 EMUs)
+		</member>
+		<member name="max_shift" type="float" setter="set_door_max_shift" getter="get_door_max_shift" default="0.0">
+			Width (or angle) of full door opening
+		</member>
+		<member name="max_shift_plug" type="float" setter="set_door_max_shift_plug" getter="get_door_max_shift_plug" default="0.0">
+			Bounce distance for Plug-type doors (sliding bounce), in meters.
+		</member>
+		<member name="mirror/max_shift" type="float" setter="set_mirror_max_shift" getter="get_mirror_max_shift" default="0.0">
+			Rotation angle for a fully unfolded mirror
+		</member>
+		<member name="mirror/vel_close" type="float" setter="set_mirror_vel_close" getter="get_mirror_vel_close" default="0.0">
+			Vehicle speed at which automatic closure of exterior mirrors occurs
+		</member>
+		<member name="needs_permit" type="bool" setter="set_door_needs_permit" getter="get_door_needs_permit" default="false">
+			Opening the doors by passengers requires permission from the train driver.
+		</member>
+		<member name="open/delay" type="float" setter="set_door_open_delay" getter="get_door_open_delay" default="0.0">
+			Door opening delay, in seconds.
+		</member>
+		<member name="open/method" type="int" setter="set_door_open_method" getter="get_door_open_method" default="0">
+			Determines method of opening doors.
+			The following values are valid:
+			Passenger - default type when no definition is provided; doors are manually closed, ignoring any remote commands
+			AutomaticCtrl - doors operate automatically
```

Comment:
Wymienione wartości nie są intami. A jeśli to są enumy, to powinien być reference do enuma. Enum będzie miał swój doc.